cmd/virt-chroot/selinux.go

```diff
@@ -31,3 +31,22 @@ func NewGetEnforceCommand() *cobra.Command {
 	}
 	return cmd
 }
+
+func RelabelCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:       "relabel",
+		Short:     "relabel a file with the given selinux label",
+		Example:   "virt-chroot selinux relabel <new-label> <file-path> ",
+		ValidArgs: nil,
+		Args:      cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			label := args[0]
+			filePath := args[1]
+
+			if err := selinux.Chcon(filePath, label, false); err != nil {
```

Comment:
Would it be possible to check the file label first, and only Chcon if oldLabel != newLabel?

Reply:
Done.